main.cc

```diff
@@ -728,6 +728,76 @@ To start the scylla server proper, simply invoke as: scylla server (or just scyl
                 mm_notifier.stop().get();
             });
 
+            supervisor::notify("starting per-shard database core");
```

Comment:
There is no much motivation in the commit message. Why is it needed? (put the question here because fsck github).